### PR TITLE
Pybind simplification

### PR DESCRIPTION
**Pull request by kutipense**
Removed unnecessary mesh dependencies since those utilities already exists in vdbfusion. There are still unnecessary parts but this version installs with no problem and tests pass.

### CODE REVIEW

File: src/vdb_to_numpy/pybind/vdb_pybind.cpp [hunk 1]
```diff
@@ -61,7 +60,6 @@ PYBIND11_MODULE(vdb_pybind, m) {
           "(8, 8, 8) grid containing the floating point values of the leaf "
           "node.",
           "grid"_a);
-    m.def("_extract_triangle_mesh", &ExtractTriangleMesh);
```
Comment (anastasiia-kornilova) [inline src/vdb_to_numpy/pybind/vdb_pybind.cpp:64]: So we will use other methods to construct mesh from tsdf, not vdb-based?

Comment (anastasiia-kornilova) [reply src/vdb_to_numpy/pybind/vdb_pybind.cpp:64]: We don't need it in data generation pipeline, but what alternatives do we have to visualize things?

**Review by anastasiia-kornilova — APPROVED** [review]

File: src/vdb_to_numpy/pybind/vdb_pybind.cpp (hunk 1 above)
Comment (kutipense) [reply src/vdb_to_numpy/pybind/vdb_pybind.cpp:64]: same function already exists in vdbfusion package, we don't need the second one here. We can still visualize it as mesh from vdb grid.

Comment (kutipense) [reply src/vdb_to_numpy/pybind/vdb_pybind.cpp:64]: https://github.com/PRBonn/vdbfusion/blob/2e4374a6b718e49b36409c89ed24aa71eecba77f/src/vdbfusion/pybind/vdbfusion_pybind.cpp#L141

**Pull request merged**